custom_news_feed/custom_news_feed.py

```diff
@@ -291,7 +317,9 @@ def addLinks(self, links):
         self.dockwidget.linksScrollArea.setVisible(len(links) > 0)
         self.dockwidget.linkSectionLabel.setVisible(len(links) > 0)
         self.dockwidget.widget = QWidget()
+        self.dockwidget.widget2 = QWidget()
```

Comment:
Q: Was ist mit `widget2` (und den anderen Controls mit `..2`) gemeint? Kann man einen aussagekräftigeren Namen dafür verwenden oder macht das wenig Sinn?

Reply:
Es handelt sich um Controls für das Archiv-Tab. Ich denke in diesem Kontext passt der Suffix 2 nicht schlecht. Habs darum so belassen